doc/index.php

```diff
@@ -26,16 +26,20 @@
 
 </ul></p>
 
-<h2>Upcoming release series</h2>
+<!-- <h2>Upcoming release series</h2>
```

Comment:
Delete, don't comment out 😄 

Perhaps link to main and call it "v5.1".

Reply:
deleted